src/basic/calendarspec.c

```diff
@@ -595,7 +601,7 @@ static int calendarspec_from_time_t(CalendarSpec *c, time_t time) {
         if (r < 0)
                 return r;
 
-        c->utc = true;
+        c->timezone = strdup("UTC");
```

Comment:
misses OOM check (see CODING_STYLE). That said, I think we should keep explicit UTC support in place, see comment above